SPEC/FILES.md

```diff
@@ -58,6 +60,8 @@ Where `data` may be:
 {
     path: '/tmp/myfile.txt', // The file path
     content: <data> // A Buffer, Readable Stream, Pull Stream or File with the contents of the file
+    mode: '0755' // optional string or integer mode to store the entry with. strings will be interpreted as a base 8 number
```

Comment:
When I think about a unix mode as a string I think “rwx” “r--“ etc. Why not use this format as our stringy mode?

Reply:
Yeah?  I have to admit I don't when setting modes, only when viewing them with `ls -l` or whatever.

In my head at least, the choice was between a literal base 10 number (e.g. `493`) or a literal base 8 number (e.g. `0755`).  Linters don't seem to like literal numbers in anything other than base 10 so that's why it's a string `'0755'` above.

It's good to be flexible though, maybe something we could add later?

Reply:
If it helps to get this stuff merged faster then sure 👍 We’ll be supporting rwx format in chmod so I reckon people will expect to be able to use it here too.

Reply:
We support Symbolic Notation in `chmod` (e.g. `a=rwx`, `ug-x`, etc) which are modifications to an existing `mode`.

I guess if you want to use it here you could apply the modification to the default mode for files/directories?